### ISSUE TITLE
Send location on Linux

### ISSUE THREAD

**Issue by b-kamalov-diginavis** [telegramdesktop/tdesktop#28557]
### Is your feature request related to a problem?

Same as https://github.com/telegramdesktop/tdesktop/issues/1193 but for Linux

### Describe the solution you'd like

I want "send location" feature to work on Linux too

### Describe alternatives you've considered

-

### Additional context

I use Telegram Desktop 5.6.3 (Flathub) on Fedora 40 (Wayland)

**Comment by john-preston** [comment]
yeah, this is possible in case webview is working for you for bots, but I didn't implement it yet 😢

**Comment by Aokromes** [comment]
implemented.
